apps/ui/components/application/ApplicationEventList.tsx

```diff
@@ -51,112 +66,204 @@ const formatDurationSeconds = (seconds: number, t: TFunction): string => {
   )}`;
 };
 
+const getDuration = (begin: number, end: number, t: TFunction): string => {
+  const beginHours = formatDurationSeconds(begin, t);
+  const endHours = formatDurationSeconds(end, t);
+  return `${beginHours} - ${endHours}`;
```

Comment:
what happens if `begin === end` why are we printing duration: `2 - 2 hours` instead of `2 hours`?